.changeset/popular-drinks-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+NFTs do not load in the gallery when displayed in list view
```

Comment:
I think we should be more explicit when writing changeset, like: `Fixing not loading NFTs in the gallery when displayed in list view`